resources/providers/awslib/s3/s3.go

```diff
@@ -25,8 +25,18 @@ import (
 )
 
 type BucketDescription struct {
```

Comment:
Consider adding struct tags.

Reply:
I'll fix this in a future PR as it requires changes to the policy code